src/middlewares/error.middleware.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-console */
+'use strict';
+
+const { ApiError } = require('../utils/api.error');
+
+const errorMiddleware = (error, req, res, next) => {
+  if (error instanceof ApiError) {
```

Comment:
Unforseen errors will be handled by this middleware. If error is not handled by me  it will throw generic error with code 500.